include/seqan3/std/span

```diff
@@ -19,9 +19,11 @@
 
 #if __has_include(<span>)
 #include <span>
+#endif
 
+#if defined(__cpp_lib_span)
 #include <seqan3/core/platform.hpp>
-#else // __has_include(<span>)
+#else // defined(__cpp_lib_span)
```

Comment:
I guess there must be a comment for the closing macro that has still __has_include?